src/st/dai/dai.h

```diff
@@ -151,37 +133,10 @@ enum OVL_EXPORT(Entities) {
     NUM_ENTITIES,
 };
 
-extern ObjInit2 OVL_EXPORT(BackgroundBlockInit)[]; // e_room_bg
-extern EInit OVL_EXPORT(EInitBreakable);
-extern EInit g_EInitObtainable;
-extern EInit g_EInitParticle;
-extern EInit g_EInitSpawner;
-extern EInit g_EInitInteractable;
-extern EInit g_EInitUnkId13;
-extern EInit g_EInitLockCamera;
-extern EInit g_EInitCommon;
-// extern EInit g_EInitDamageNum;
-extern EInit g_EInitEnvironment;
-extern EInit g_EInitConfessionalGhost;
-extern EInit g_EInitConfessionalBlades;
-extern EInit g_EInitCornerGuard;
-extern EInit g_EInitCornerGuardAttack;
-extern EInit g_EInitBonePillarSkull;
-extern EInit g_EInitBonePillarParts;
-extern EInit g_EInitBonePillarFireBreath;
-extern EInit g_EInitBonePillarSpikeBall;
-extern EInit g_EInitBoneHalberd;
-extern EInit g_EInitBoneHalberdAttack;
-extern EInit g_EInitWingedGuard;
-extern EInit g_EInitWingedGuardParts;
-extern EInit g_EInitBat;
-extern EInit g_EInitBlueRaven;
-extern EInit g_EInitBlackCrow;
-extern EInit g_EInitSkelerang;
-extern EInit g_EInitSkelerangBoomerang;
-extern EInit g_EInitHuntingGirl;
-extern EInit g_EInitSpectralSword;
-extern EInit g_EInitSpectralSwordAura;
-extern EInit g_EInitSpectralSwordWeapon;
+// These are used by both e_chair and e_confessional
+#define CONFESSIONAL_GHOST_PRIEST 0
+#define CONFESSIONAL_GHOST_PARISHIONER 1
+#define CONFESSIONAL_GHOST_BAD 0x100
+#define CONFESSIONAL_GHOST_READY 1
```

Comment:
I know these are being shifted around, but is there any reason to not use an `enum`?

Reply:
In looking at this, I don't think it really benefits from being in dai.h anyway.  I ended up changing this to an enum and putting it in the respective files.